extensions/debezium-sqlserver/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,13 @@
+=== Encryption
+
+SQL server may require encryption. In that case `quarkus.ssl.native=false` parameter has to be added during execution.
+See https://quarkus.io/guides/native-and-ssl for more details.
+
+=== Integration testing
```

Comment:
Maybe move this section into a dedicated README in the debezium itest module. It's not really required information for the most users.